elasticsearch/probe/readiness.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright 2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO: we need to re-use code that is currently found in ./run.sh
+ES_REST_BASEURL=https://localhost:9200
+secret_dir=/etc/elasticsearch/secret
+max_time=300
```

Comment:
I think the `readinessProbe` can be [configured with `timeoutSeconds`](https://docs.openshift.org/latest/dev_guide/application_health.html):

> How long to wait for the probe to finish (default: 1). If this time is exceeded, OpenShift Origin considers the probe to have failed.

But may be we should still try to terminate the curl command if it runs to long regardless Kubernetes timeouting the probe test? May be we should just make sure that the curl timeout is not set shorter than the probe `timeoutSeconds`?